src/main/java/org/onedatashare/server/controller/UserController.java

```diff
@@ -59,6 +59,15 @@ public Object performAction(@RequestHeader HttpHeaders headers, @RequestBody Use
         return null;
     }
   }
+  @PutMapping
+  public Object putAction(@RequestHeader HttpHeaders headers, @RequestBody UserAction userAction){
+    switch(userAction.action) {
+      case "enableOrDisableAdmin":
```

Comment:
For clarity, can we set this case as 'updateAdminRights'?

Reply:
Incorporated.